Core/scripts/dirac-install.py

```diff
@@ -843,11 +843,17 @@ def downloadAndExtractTarball( tarsURL, pkgName, pkgVer, checkHash = True, cache
   tarName = "%s-%s.tar.gz" % ( pkgName, pkgVer )
   tarPath = os.path.join( cliParams.targetPath, tarName )
   tarFileURL = "%s/%s" % ( tarsURL, tarName )
+  tarFileCVMFS = "/cvmfs/dirac.egi.eu/installSource/%s" % tarName
```

Comment:
In LHCb we have our pilot command for installing LHCbDIRAC (from CVMFS). Our case is probably more complex that yours, but still this can also be a possibility for you (that you can have today).